### ISSUE TITLE
CreateFramesetter crashes

### ISSUE THREAD

**Issue by Rich2k** [jasarien/CoreTextHyperlinkView#7]
I love the work on this but I'm getting quite a few crash reports related to two areas

1   AppName                           0x001956c3 -[JSCoreTextView createFramesetter](JSCoreTextView.m:546)
2   AppName                           0x00195789 -[JSCoreTextView drawInContext:bounds:](JSCoreTextView.m:570)
3   AppName                           0x00195f59 -[JSCoreTextView drawRect:](JSCoreTextView.m:703)

and

0   CoreFoundation                      0x35e6b9f0 CFEqual + 260
1   CoreFoundation                      0x35ed3b67 CFAttributedStringSetAttribute + 183
2   AppName                          0x0018d5cb -[JSCoreTextView createFramesetter](JSCoreTextView.m:546)
3   AppName                          0x0018d6e9 -[JSCoreTextView drawInContext:bounds:](JSCoreTextView.m:570)
4   AppName                           0x0018de79 -[JSCoreTextView drawRect:](JSCoreTextView.m:703)

Any ideas?

**Comment by jasarien** [comment]
Do you have any information on the situations in which these crashes occur? iOS version, device model, etc, etc.

**Comment by Rich2k** [comment]
I suppose the difficulty is that I can't reproduce it myself.  I'm using in a custom UITableViewCell. Setting the TextView in the cell creation code and then modifying the text and height in the update code.

From reading some posts from various sources I made a few very minor changes and it seems to have gone away (well the complaints have anyway!)

Specifically though I changed CFAttributedStringSetAttribute calls for the font name, color and paragraph style to a single call of

CFAttributedStringSetAttributes using an NSDictionary.
